crates/lib/src/imgstorage.rs

```diff
@@ -104,7 +104,7 @@ fn bind_storage_roots(cmd: &mut Command, storage_root: &Dir, run_root: &Dir) ->
                 Mode::empty(),
             )?;
             rustix::process::fchdir(&storage_root)?;
-            rustix::thread::unshare(rustix::thread::UnshareFlags::NEWNS)?;
+            rustix::thread::unshare_unsafe(rustix::thread::UnshareFlags::NEWNS)?;
```

Comment:
TODO: Investigate this further.  Is this the right thing to do?  It was recommended in the crate, but we should try and avoid unsafe code.

Reply:
It appears `unshare` itself is considered unsafe, and this name change simply clarifies it as such.
https://github.com/bytecodealliance/rustix/pull/1482